clients/src/main/java/org/apache/kafka/clients/admin/KafkaAdminClient.java

```diff
@@ -553,7 +558,11 @@ private KafkaAdminClient(AdminClientConfig config,
         this.timeoutProcessorFactory = (timeoutProcessorFactory == null) ?
             new TimeoutProcessorFactory() : timeoutProcessorFactory;
         this.maxRetries = config.getInt(AdminClientConfig.RETRIES_CONFIG);
-        this.retryBackoffMs = config.getLong(AdminClientConfig.RETRY_BACKOFF_MS_CONFIG);
+        this.retryBackoff = new ExponentialBackoff(
```

Comment:
I think in the KIP we had stated that if `retry.backoff.ms` was set to be greater than `retry.backoff.max.ms`, we will log a warning and state that `retry.backoff.max.ms` will be used as the static backoff. It would be prudent to put this `WARN` level log in all the clients upon startup. Also, we need to account for this case in `ExponentialBackoff.java`, as I don't think it's accounted for in the current implementation.

Reply:
Yes. We can do something similar to `postProcessReconnectBackoffConfigs`